drivers/dac/ad5758/ad5758.h

```diff
@@ -273,17 +276,17 @@ enum ad5758_dc_dc_mode {
 };
 
 enum ad5758_dig_diag_flags {
-	DIAG_SPI_CRC_ERR,
-	DIAG_SLIPBIT_ERR,
-	DIAG_SCLK_COUNT_ERR,
+	DIAG_SPI_CRC_ERR = 0,
```

Comment:
nitpick: this is not necessary. Enum will do this itself.

Reply:
This is written in order to respect coding standard.